File: support_tools/swigwin-3.0.12/Examples/test-suite/python/enums_runme.py

import _enums

_enums.bar2(1)
_enums.bar3(1)
_enums.bar1(1)

if _enums.cvar.enumInstance != 2:
    raise RuntimeError

if _enums.cvar.Slap != 10:
    raise RuntimeError

if _enums.cvar.Mine != 11:
    raise RuntimeError

if _enums.cvar.Thigh != 12:
    raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/enums_runme.go
package main

import "./enums"

func main() {
	enums.Bar2(1)
	enums.Bar3(1)
	enums.Bar1(1)

	if enums.GetEnumInstance() != 2 {
		panic(0)
	}

	if enums.GetSlap() != 10 {
		panic(0)
	}

	if enums.GetMine() != 11 {
		panic(0)
	}

	if enums.GetThigh() != 12 {
		panic(0)
	}
}
